source/simulator/parameters.cc

```diff
@@ -2167,73 +2167,6 @@ namespace aspect
     }
     prm.leave_subsection();
 
-    prm.enter_subsection ("Boundary traction model");
```

Comment:
this piece of code is completely useless.

Reply:
Are you saying that because it never seems to set variables that are not local to the scope? I get that for everything after the assertion, but the assertion seems useful to me.

Reply:
This code used to set up boundary traction plugins but that has been moved into the Manager class. The Assert statement here is the only useful bit (none of the other variables are used for anything), but the same one is also checked here:
https://github.com/geodynamics/aspect/blob/590e4e2d9034fa38c908b50d94525de356bedb1a/source/boundary_traction/interface.cc#L176